Rubberduck.VBEditor.VBA/Rubberduck.VBEditor.VBA.csproj

```diff
@@ -112,7 +112,7 @@
       <EmbedInteropTypes>False</EmbedInteropTypes>
     </Reference>
     <Reference Include="stdole, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <EmbedInteropTypes>True</EmbedInteropTypes>
+      <EmbedInteropTypes>False</EmbedInteropTypes>
```

Comment:
Do we want that? I thought we should be embedding to avoid problems w/ versioning?

Reply:
Hmm I toggled that in response to a build warning.